README.md

```diff
@@ -1,2 +1,167 @@
+[downloads-image]:http://img.shields.io/npm/dm/react-idm-wallet.svg
+[npm-image]:http://img.shields.io/npm/v/react-idm-wallet.svg
+[travis-url]:https://travis-ci.org/ipfs-shipyard/react-idm-wallet
+[travis-image]:http://img.shields.io/travis/ipfs-shipyard/react-idm-wallet/master.svg
+[codecov-url]:https://codecov.io/gh/ipfs-shipyard/react-idm-wallet
+[codecov-image]:https://img.shields.io/codecov/c/github/ipfs-shipyard/react-idm-wallet/master.svg
+[david-dm-url]:https://david-dm.org/ipfs-shipyard/react-idm-wallet
+[david-dm-image]:https://img.shields.io/david/ipfs-shipyard/react-idm-wallet.svg
+[david-dm-dev-url]:https://david-dm.org/ipfs-shipyard/react-idm-wallet?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/ipfs-shipyard/react-idm-wallet.svg
+
+React bindings for the [reference implementation](https://npmjs.org/package/idm-wallet) of the IDM Wallet in JavaScript.
+
+
+## Installation
+
+```sh
+$ npm install react-idm-wallet idm-wallet
+```
+
+You must also install `idm-wallet` as this module has a peer-depedency on it.
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+
+## Usage
+
+First, wrap your application in `<IdmWalletProvider>`:
+
+```js
+import React from 'react';
+import ReactDOM from 'react-dom';
+import createIdmWallet from 'idm-wallet';
+import { IdmWalletProvider } from 'react-idm-wallet';
+
+createIdmWallet()
+.then(renderApp)
+.catch((err) => console.error(err));
+
+const renderApp = (idmWallet) => {
+    ReactDOM.render(
+        <IdmWalletProvider idmWallet={ idmWallet }>
+            <App />
+        </IdmWalletProvider>,
+        document.getElementById('root')
+    );
+};
+```
+
+Then, you may use `connectIdmWallet` to connect a component to a IDM Wallet:
+
+```js
+// App.js
+import React from 'react';
+import { connectIdmWallet } from 'react-idm-wallet';
+
+const App = ({ locked, onLock }) => (
+    <div>
+        <p>Locked: { locked }</p>
+        <button onClick={ onLock }>Lock wallet</button>
+    </div>
+);
+
+const createMapWalletToProps = (idmWallet) => {
+    const onLocked = () => idmWallet.locker.lock();
+
+    return () => {
+        locked: idmWallet.locker.isLocked(),
+        onLock,
+    });
+};
+
+export connectIdmWallet(createMapWalletToProps)(App);
+```
+
+
+## API
+
+- [`<IdmWalletProvider>`](#idmwalletprovider)
+- [`connectIdmWallet(createMapWalletToProps, [options])`](#connectidmwalletcreatemapwallettoprops-options)
+
+### IdmWalletProvider
+
+The `<IdmWalletProvider>` makes a IDM Wallet available to any nested components that have been wrapped in the `connectIdmWallet()` function.
+
+Since any React component in an app can be connected, most applications will render a <IdmWalletProvider> at the top level, with the entire app's component tree inside of it. You can't use a connected component unless it is nested inside of a `<IdmWalletProvider>`.
```

Comment:
You are missing back-ticks around  &lt;IdmWalletProvider&gt; 

This: 
```
Since any React component in an app can be connected, most applications will render a <IdmWalletProvider> at the top level (...)
```

Must change to this: 
```
Since any React component in an app can be connected, most applications will render a `<IdmWalletProvider>` at the top level (...)
```

Otherwise, &lt; and &gt; will create links and you will see nothing on markdown since IdmWalletProvider is not a valid link.

Reply:
Nice catch, fixed.